process/coordinator/processProposal.go

```diff
@@ -23,17 +23,20 @@ func (tc *transactionCoordinator) CreateMbsCrossShardDstMe(
 	miniBlocksAndHashes := make([]block.MiniblockAndHash, 0)
 	numTransactions := uint32(0)
 	shouldSkipShard := make(map[uint32]bool)
-	// TODO: init the gas estimation per added mbs counter
+
+	tc.gasComputation.Reset()
```

Comment:
more like a double check for me: so between this `Reset` and the end of this function we are able to call all the `gasComputation` related functions?

Reply:
and the next call will use a "new clean" gas computation for the remaining miniblocks if any

Reply:
exactly, each selection process should use a clean state

Reply:
moved as this was not the right place for it